database/database.go

```diff
@@ -0,0 +1,115 @@
+package database
+
+import (
+	"dashboard/model"
+	"database/sql"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+var db *sql.DB
+
+//InitialiseDb to start db server
+func InitialiseDb() error {
+	var err error
+	db, err = sql.Open("mysql", "root:mysql1234@tcp(localhost:3306)/dashboard")
+	return err
+}
+
+//CloseDb to close DB connection
+func CloseDb() {
+	db.Close()
+}
+
+//GetUserByUsername func to get user by username
+func GetUserByUsername(username string) (*model.UserProfile, error) {
+	var err error
+	var profile *model.UserProfile
+	stmtOut, err := db.Prepare("SELECT * FROM user_profile WHERE username  = ? ")
+	if err != nil {
+		return nil, err

```

Comment:
log error before returning it, but use log.Error instead of log.Panic. Read about difference between them.